docs/messaging/gcp-pubsub.md

```diff
@@ -16,7 +16,14 @@ For Google Cloud Pub/Sub, the following additional attributes are defined:
 <!-- semconv messaging.gcp_pubsub(full,tag=tech-specific-gcp-pubsub) -->
 | Attribute  | Type | Description  | Examples  | Requirement Level |
 |---|---|---|---|---|
+| [`messaging.gcp_pubsub.ack_deadline_seconds`](../attributes-registry/messaging.md) | int | The ack deadline set for the modify ack deadline request | `10` | Recommended |
+| [`messaging.gcp_pubsub.message.ack_id`](../attributes-registry/messaging.md) | string | The ack id for a given message. | `ack_id` | Recommended: [1] |
+| [`messaging.gcp_pubsub.message.delivery_attempt`](../attributes-registry/messaging.md) | int | The delivery attempt for a given message. | `2` | Recommended: [2] |
```

Comment:
For batch operations, can these attributes also appear on links?

Reply:
That's a good point. Yes, I think that would be helpful.

I updated the conditional requirement to say only if the span represents an operation on a single message, that way it could be added as an attribute to the "Receive" span links.